bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/single_VLQ/NWA/TprimeBToTZ_LH/TprimeBToTZ_LH_narrowLO_M700/TprimeBToTZ_LH_narrowLO_M700_madspin_card.dat

```diff
@@ -0,0 +1,11 @@
+set ms_dir ./madspingrid
+set max_running_process 1
+
+set Nevents_for_max_weight 250 # number of events for the estimate of the max. weight
+set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+
+decay t > w+ b, w+ > all all
+decay t~ > w- b~, w- > all all
+decay z > all all
+
+launch
```

Comment:
is there a specific need for using madspin, if not, use pythia for all particles decay would be fast.

Reply:
It was added to make sure of propagating the information of the left-(or right for the other samples) handed production on angular observables in top quark decays.

Reply:
please have a check on the number of file in the gridpack, several ten thousands or even larger number of files are not encouraged in production. If this is the case, you may need to separate the process into several samples according to the decay mode.

Reply:
What do you mean for the number of file in gridpack?

Reply:
just count all the files after you untar the gridpack

Reply:
Ok. I checked the number of files, it is higher then ten thousands.
One possible solution would be to leave Z decays to Pythia instead of doing it with madspin, leaving it just for top quark decays. This would reduce the #files below 10k. Would such modification work for you? Can you also confirm all decays of the Z boson are included in the pythia step?

Reply:
~10K should be fine.